ui/src/kapacitor/components/RuleMessageText.tsx

```diff
@@ -7,16 +9,40 @@ interface Props {
   updateMessage: (e: ChangeEvent<HTMLTextAreaElement>) => void
 }
 
-const RuleMessageText: SFC<Props> = ({rule, updateMessage}) => (
-  <div className="rule-builder--message">
-    <textarea
-      className="form-control input-sm form-malachite monotype"
-      onChange={updateMessage}
-      placeholder="Example: {{ .ID }} is {{ .Level }} value: {{ index .Fields &quot;value&quot; }}"
-      value={rule.message}
-      spellCheck={false}
-    />
-  </div>
-)
+const RuleMessageText: SFC<Props> = ({rule, updateMessage}) => {
```

Comment:
I know this started out as an SFC, but I think this would be easier to read and extend if converted to a class with getters for `textAreaClass`, `isValid`, and removing the ternaries.